### PR TITLE
Forward the request when saving the form

### PR DESCRIPTION
**Pull request by felippem**
Hello. I would like to forward the request to the instance of ConstanceForm, when calling the save method. Help me understand if it is necessary to update any tests.

### CODE REVIEW

**Comment by felippem** [comment]
@Mogost @jezdez @Mogost Can you please review this?

**Comment by jezdez** [comment]
@felippem Yes, the tests need to be updated, but I'm also curious for the use case?

**Comment by felippem** [comment]
> @felippem Yes, the tests need to be updated, but I'm also curious for the use case?

@jezdez I'd like to get the request.user.has_perm when the form loads, after it's posted.  I have a custom form, which inherits Constance's original implementation.

**Comment by felippem** [comment]
> > @felippem Yes, the tests need to be updated, but I'm also curious for the use case?
> 
> @jezdez I'd like to get the request.user.has_perm when the form loads, after it's posted.  I have a custom form, which inherits Constance's original implementation.

Hi @jezdez ! Did you understand my use case?

**Review by Mogost — APPROVED** [review]

**Comment by jezdez** [comment]
@felippem I get it now, but this is actually a regression, since there is a [permission check in the `ConstanceForm.__init__` method](https://github.com/jazzband/django-constance/blob/02c5fd5011d8557f99af196bc9f73fa1df001eb4/constance/admin.py#L105) that is currently not run since the request isn't passed. Since the tests don't fail, it seems as if this line of code isn't accurately tested. Please add/update a test case for this.

**Comment by jezdez** [comment]
This was an oversight in https://github.com/jazzband/django-constance/pull/393.

**Comment by felippem** [comment]
> @felippem I get it now, but this is actually a regression, since there is a [permission check in the `ConstanceForm.__init__` method](https://github.com/jazzband/django-constance/blob/02c5fd5011d8557f99af196bc9f73fa1df001eb4/constance/admin.py#L105) that is currently not run for POST requests since the request isn't passed. Since the tests don't fail, it seems as if this line of code isn't accurately tested. Please add/update a test case for this.

So I'll try to do that soon.  Thanks for answering.

**Comment by felippem** [comment]
> > @felippem I get it now, but this is actually a regression, since there is a [permission check in the `ConstanceForm.__init__` method](https://github.com/jazzband/django-constance/blob/02c5fd5011d8557f99af196bc9f73fa1df001eb4/constance/admin.py#L105) that is currently not run for POST requests since the request isn't passed. Since the tests don't fail, it seems as if this line of code isn't accurately tested. Please add/update a test case for this.
> 
> So I'll try to do that soon. Thanks for answering.

@jezdez Done. Can you review please?

File: docs/changes.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 ---------
 
+2.10.0 (2022-10-08)
+~~~~~~~~~~~~~~~~~~
+
+* Regression fix (#393) - forward the request when saving the form
+
```
Comment (felippem) [inline docs/changes.rst:8]: @jezdez What do you think about this?

**Comment by felippem** [comment]
Thank you @jezdez. Have an idea about the next release?

**Comment by felippem** [comment]
@jezdez can I help with anything else?

**Review by camilonova — APPROVED** [review]

**Comment by camilonova** [comment]
Thank you

**Comment by felippem** [comment]
Hi, @camilonova! Do you have a date for the next release?

**Pull request merged**